core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorSpec.scala

```diff
@@ -2368,8 +2369,8 @@ class SubmissionMonitorSpec(_system: ActorSystem)
       super.save() flatMap { _ =>
         withWorkspaceContext(workspace) { ctx =>
           DBIO.seq(
-            entityQuery.save(
-              ctx,
+            compactEntityQuery.batchWriteEntities(
+              ctx.workspaceIdAsUUID,
               Seq(
                 legacyTestData.aliquot1,
```

Comment:
Same here and in the rest of `ManySubmissionsTestData` - switch out `legacyTestData` with `testData`.  For just getting entity name and type it won't matter, but it will for workspace id and maybe some other things, and someday when we get rid of `legacyTestData` it'll be a bit easier.